aten/src/ATen/QTensorImpl.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <c10/core/TensorImpl.h>
+#include <ATen/Quantizer.h>
+#include <c10/util/Exception.h>
+
+namespace at {
+
+struct CAFFE2_API QTensorImpl : public c10::TensorImpl {
```

Comment:
@gchanan Shall we let this in or wait for OpaqueTensorImpl to land first?

Reply:
I'd prefer to land it faster if it's the only contention point and change in a separate PR

Reply:
I don't see why this would use OpaqueTensorImpl.  That's really for stuff without a Storage.